File: Groove.SP/groove-sp-web/src/app/core/models/shipments/shipment-contact.model.ts
import { ModelBase } from '../model-base.model';

export interface ShipmentContactModel extends ModelBase {
    id: number;
    shipmentId: number;
    organizationId: number;
    organizationRole: string;
    companyName: string;
    address: string;
    contactName: string;
    contactNumber: string;
    contactEmail: string;
}

File: Groove.SP/Groove.SP.Core/Entities/ShipmentContactModel.cs
namespace Groove.SP.Core.Entities
{
    public class ShipmentContactModel : Entity
    {
        public long Id { get; set; }

        public long ShipmentId { get; set; }

        public long OrganizationId { get; set; }

        public string OrganizationRole { get; set; }

        public string CompanyName { get; set; }

        public string Address { get; set; }

        public string ContactName { get; set; }

        public string ContactNumber { get; set; }

        public string ContactEmail { get; set; }



        public virtual ShipmentModel Shipment { get; set; }
    }
}
